app.rb

```diff
@@ -1,9 +1,38 @@
 require 'sinatra/base'
+require 'sinatra/reloader'
+require_relative './lib/peep'
 
 class Chitter < Sinatra::Base
+
+  configure :development do
+    register Sinatra::Reloader
+  end
+
+  get '/' do
```

Comment:
It's a bit strange to have a route redirecting to another route, since we don't ask for a homepage, you can change this so we're just getting all peeps.